Library/Homebrew/compat/md5.rb

```diff
@@ -1,26 +1,35 @@
 class Formula
   def self.md5(val)
+    # Don't throw an error at load time, just install time.
     stable.md5(val)
   end
 end
 
 class SoftwareSpec
   def md5(val)
+    odie <<-EOS.undent
+      Using MD5 has been dropped for security reasons.
+      Please switch to SHA-256.
+    EOS
```

Comment:
I see. My impression was that formula loading in general was guarded by some exception handling, but that's apparently only happening in `brew readall`, `brew update-report`, and maybe a few other places. I wonder whether this is something that could (and should) be addressed in `Formulary` …

Though I'm not sure if killing the process is a real issue. The greater issue in my opinion that there's no indication which formula is causing the error, unless `--debug` is also supplied to the invocation that failed.